frontend/packages/noobaa-storage-plugin/src/components/resource-providers-card/resource-providers-card-body.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ArrowCircleDownIcon } from '@patternfly/react-icons';
+import { LoadingInline } from '@console/internal/components/utils';
+
+const ResourceProvidersRowStatus: React.FC<ResourceProvidersRowStatusProps> = React.memo(
+  ({ status }) => (
+    <div className="co-resource-providers-card__row-status">
```

Comment:
all `noobaa` styles should be prefixed with `noobaa` or maybe `nb` instead of `co`. Can you also update other noobaa PRs with the following change ?